ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/FloorKudf.java

```diff
@@ -14,16 +14,45 @@
 
 package io.confluent.ksql.function.udf.math;
 
-import io.confluent.ksql.function.KsqlFunctionException;
-import io.confluent.ksql.function.udf.Kudf;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
 
-public class FloorKudf implements Kudf {
+@UdfDescription(name = FloorKudf.NAME, version = "Confluent",
+    description = "Returns the floor of a value.")
+public class FloorKudf {
+  static final String NAME = "FLOOR";
 
-  @Override
-  public Object evaluate(final Object... args) {
-    if (args.length != 1) {
-      throw new KsqlFunctionException("Floor udf should have one input argument.");
+  @Udf(description = "Returns the floor of a value.")
```

Comment:
nit: I don't think you need to duplicate the description in the `@Udf` parameter. I *think* if its left empty it will default to the one from `@UdfDescription`.   Try it!

If it is required - then put the description in a constant.